conans/paths/package_layouts/package_cache_layout.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, base_folder, ref, short_paths, no_lock):
         assert isinstance(ref, ConanFileReference)
         self._ref = ref
         self._base_folder = os.path.normpath(base_folder)
-        self._short_paths = short_paths
+        self.short_paths = short_paths
```

Comment:
Keep it _hidden_, please.

Probably we will need a property to access this value and handle some logic as there are two different use cases: sometimes the value is provided from outside (we have the `conanfile` already loaded and we know the value, we are populating the layout and saving to the cache), but sometimes we will be retrieving an existing package and this value should be read from the stored `conanfile`.